### ISSUE TITLE
Create CRAN/Bioconductor package?

### ISSUE THREAD

**Issue by apeltzer** [kkdey/aRchaic#1]
Hi! 
would you be willing to create a CRAN/Bioconductor package for this? Would be great to have easier installation possibilities for users!

**Comment by kkdey** [comment]
Yes, that is in our long-term pipeline. We are working on having the dependencies on Bioc first. Also, since the package is in a bit of a flux, with users suggesting new functions to add to the pipeline, we are holding off on the submission right now. But, definitely we want to have it on Bioc in near future.
